monitor/src/main/java/edu/asu/diging/monitor/core/cron/impl/AppChecker.java

```diff
@@ -93,6 +93,11 @@ public void checkApp(IApp app) {
 			notificationManager.sendNotificationEmails(app, lastTest.getStatus());
 		}
 	}
+	
```

Comment:
let's change the `checkApp` method so that it saves the apptest after it has been created and set's it's status to "in progress" (and then set it to ok or error once the result is determined). This way the UI can show that a test is in progress.